yt/yt/tests/integration/node/test_layers.py

```diff
@@ -557,6 +563,57 @@ def test_wrong_tag_document_type(self):
         with raises_yt_error(f'Tags document "{TestDockerImage.TAG_DOCUMENT_PATH}" is not a map'):
             self.run_map(f"{TestDockerImage.IMAGE}:tag")
 
+    @authors("khlebnikov")
+    @pytest.mark.timeout(180)
+    def test_invalid_external_image(self):
```

Comment:
Maybe this test was not checked?

Looks like it is missing a call to `self.create_tables()`:
```
yt/yt/tests/integration/node/test_layers.py:570: in test_invalid_external_image
    self.run_map(self.INVALID_EXTERNAL_IMAGE)
yt/yt/tests/integration/node/test_layers.py:493: in run_map
    map(
yt/yt/tests/library/yt_commands.py:1660: in map
    return start_op("map", **kwargs)
yt/yt/tests/library/yt_commands.py:1594: in start_op
    operation.track()
yt/yt/tests/library/yt_commands.py:1465: in track
    raise self.get_error()
E   yt.common.YtError: Operation has failed to initialize
E       Error locking input tables
E           Error communicating with master
E               Error resolving path //tmp/input_table
E                   Node //tmp has no child with key "input_table"
```